app/static/js/scratch_datatables.js

```diff
@@ -323,8 +358,12 @@ function initDataTables() {
     }
   }
 
-  let table = $(".table table, .table_detail table");
-  return table.DataTable(tableOptions[table.attr("id")]);
+  let tables = $(".table table, .table_detail table");
+
+  return $.each(tables, function(index, value){
+    return $(value).DataTable(tableOptions[$(value).attr("id")])
+  })
```

Comment:
I was thinking of something like this:
```
$(".table table, .table_detail table").each(function(){
  $(this).DataTable(tableOptions[$(this).attr("id")])
})
```
From the docs: https://api.jquery.com/each/

Initially I was returning `table.DataTable(tableOptions[table.attr("id")]);` in the `initDataTables` function because I was assuming that there would be a single DataTable per page. If this is not the case, I think there is no point trying to return multiple DataTable objects. We're not using the return value of this function anywhere anyway.